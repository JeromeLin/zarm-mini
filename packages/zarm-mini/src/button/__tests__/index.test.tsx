import React from 'react';
import { render } from '@testing-library/react';
import Button from '../index';

describe('Button', () => {
  it('renders correctly', () => {
    const { container } = render(<Button>foo</Button>);
    expect(container).toMatchSnapshot();
  });

});
