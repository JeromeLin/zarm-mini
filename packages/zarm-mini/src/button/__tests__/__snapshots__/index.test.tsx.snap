// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<div>
  <taro-view-core
    class="za-button za-button--default za-button--md za-button--radius"
  >
    <taro-button-core
      class="za-button--base"
      htmltype="button"
    />
    <taro-view-core
      class="za-button__content"
    >
      <taro-text-core>
        foo
      </taro-text-core>
    </taro-view-core>
  </taro-view-core>
</div>
`;
